experimental/PropertyDDS/packages/property-common/src/test/datastructures/collection.spec.ts

```diff
@@ -439,65 +411,50 @@ describe('collection', function () {
         done();
     });
 
-    it('should copy the collection', function (done) {
-        var collection = new Collection();
+    it("should copy the collection", function(done) {
+        const collection = new Collection();
 
-        var objectToAdd = {
-            item1: 'test',
+        const objectToAdd = {
+            item1: "test",
             item2: true,
-            item3: 23
+            item3: 23,
         };
 
         collection.bulkAdd(objectToAdd);
 
-        var collection2 = new Collection();
+        const collection2 = new Collection();
 
         collection2.copy(collection);
 
-        expect(collection2.items['item1']).to.equal(objectToAdd['item1']);
-        expect(collection2.items['item2']).to.equal(objectToAdd['item2']);
-        expect(collection2.items['item3']).to.equal(objectToAdd['item3']);
-
-        done();
-    });
-
-    it('should respect the type of Collection given', function (done) {
```

Comment:
I assume this test has been removed, because the type check had been removed earlier. Why did you remove the type check?